erpnext/patches/v8_1/remove_quotations_route_in_sidebar.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doctype("Portal Settings")
+
+	frappe.db.sql("""
+		delete from
+			`tabPortal Menu Item`
+		where
+			route = '/quotations' and title = 'Supplier Quotation'""")
+
+	frappe.db.sql("""
```

Comment:
Hi @mainul94 , I have made the changes requested. Also, I have edited "Quotation" a little to view page instead of print and added "has_wesite_permission" for the same. Thanks!

![quotation view](https://user-images.githubusercontent.com/21003054/28451994-25191ae6-6e23-11e7-9d97-b86b27479c78.png)